README.md

```diff
@@ -728,6 +728,30 @@ In either case:
       end
     ```
 
+* <a name="unless-with-comparator"></a>Avoid `unless` with comparators if you can use `if` with an opposing comparator.<sup>[[link](#unless-with-comparator)]</sup>
```

Comment:
do we need a rationale? could it be as simple as arbitrarily picking a standard? my rationale is that it's more readable, but that's an eye of the beholder style argument